# loads a plugin
string sessionname
int32 sessionid

# absolute path of plugin recommended
string filename
---
uint8 success
